umka/atlas.um

```
@@ -59,21 +63,25 @@ const (
 )
 //~~
 
-fn umth_atlas_pack(a: ^Atlas, images: ^[]image.Image, strategy: int)
+fn umth_atlas_pack(a: ^Atlas, images: ^[]image.Image, strategy: int): th.ErrCode
 
 //~~fn pack
 // Packs an array of images into an atlas
 fn pack*(images: []image.Image, strategy: int): Atlas {
```

Yeah I missed that one, the error is discarded here